__tests__/app.test.js

```diff
@@ -73,4 +72,65 @@ describe('GET /api/reviews/:review_id', () => {
             expect(res.body.msg).toBe('invalid endpoint syntax')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/reviews', () => {
+    test('GET - STATUS: 200 - responds with a reviews array of all review objects with their relevant properties, excluding review_body', () => {
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((res) => {
+            expect(res.body.reviews.length).toBe(13);
+            expect(Array.isArray(res.body.reviews)).toBe(true);
+            res.body.reviews.forEach((review) => {
+                expect(typeof review.owner).toBe('string');
+                expect(typeof review.title).toBe('string');
+                expect(typeof review.review_id).toBe('number');
+                expect(typeof review.category).toBe('string');
+                expect(typeof review.review_img_url).toBe('string');
+                expect(typeof review.created_at).toBe('string');
+                expect(typeof review.votes).toBe('number');
+                expect(typeof review.designer).toBe('string');
+                expect(review).not.toHaveProperty('review_body');
+            })
+    })
+})
+        test('GET - STATUS: 200 - the response should include a comment count which is the total count of all the comments with this review_id ', () => {
+    return request(app)
+    .get('/api/reviews')
+    .expect(200)
+    .then((res) => {
+        res.body.reviews.forEach((review) => {
+        expect(typeof review.comment_count).toBe('number')
+        expect(res.body.reviews[5].comment_count).toBe(3)
+        expect(res.body.reviews[3].comment_count).toBe(0)
+    })
```

Comment:
This can be added to the first test so that the comment_count can be any number to keep your testing suite a little dryer.